build.gradle

```diff
@@ -105,7 +105,7 @@ test {
 wrapper { gradleVersion = '6.5' }
```

Comment:
I don't think it's in-sync with the actual Gradle wrapper version. I'm also not a fan of defining the wrapper in the `build.gradle`. As the `wrapper update` will not change this value, I think it's misleading. I'd vote for removing it. Or do you have any good reasons to keep it?

Reply:
No, I did not see it.

Reply:
removed